### PR TITLE
ENG-6991 Privacy and code signing

### PR DESCRIPTION
**Pull request by cometman**

### CODE REVIEW

**Review by kishisaka-nid — APPROVED** [review]
FWIW, LGTM! 🚢

**Review by priya-xavier — APPROVED** [review]

File: PrivacyInformation.xcprivacy [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+		<dict>
+			<key>NSPrivacyCollectedDataType</key>
+			<string></string>
+			<key>NSPrivacyCollectedDataTypeLinked</key>
+			<false/>
+			<key>NSPrivacyCollectedDataTypeTracking</key>
+			<false/>
+			<key>NSPrivacyCollectedDataTypePurposes</key>
+			<array>
+				<string></string>
```
Comment (kevinsites) [inline PrivacyInformation.xcprivacy:16]: Does this need a description?

File: NeuroID.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1094,8 +1123,9 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 153C967454B5320D895ADE13 /* Pods-SDKTest.debug.xcconfig */;
 			buildSettings = {
+				CODE_SIGN_IDENTITY = "Apple Distribution: NEURO-ID, INC. (9B7FQ8D7B8)";
```
Comment (kevinsites) [inline NeuroID.xcodeproj/project.pbxproj:1126]: Is this housed in fast lane and s3 or created on your local machine or?

File: NeuroID.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1057,11 +1086,11 @@
 			baseConfigurationReference = 91F02E629A22E34E06D2AF87 /* Pods-NeuroID.release.xcconfig */;
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_IDENTITY = "";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Automatic;
```
Comment (kevinsites) [inline NeuroID.xcodeproj/project.pbxproj:1090]: Is this managed then by apple or on your local machine or?

File: PrivacyInformation.xcprivacy (hunk 1 above)
Comment (cometman) [reply PrivacyInformation.xcprivacy]: This was an extra file. Removed.

File: NeuroID.xcodeproj/project.pbxproj (hunk 2 above)
Comment (cometman) [reply NeuroID.xcodeproj/project.pbxproj:1122]: All in fastlane/s3. https://s3.console.aws.amazon.com/s3/buckets/fastlane20230807215426085400000001?region=us-east-1&bucketType=general&prefix=9B7FQ8D7B8/&showversions=false

**Review by kevinsites — COMMENTED** [review]
Pulled and everything built on my local machine after I signed into the Org Apple Account. I am assuming you need this merged to test on our runner? 

Also I ran pod install on the sandbox with my local SDK Copy and I didn't see the privacy manifest included in the pod, is that expected or should it be included?

**Review by kevinsites — APPROVED** [review]

**Review by priya-xavier — DISMISSED** [review]

**Review by kishisaka-nid — DISMISSED** [review]

**Review by priya-xavier — APPROVED** [review]

File: NeuroID.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -809,11 +830,11 @@
 			baseConfigurationReference = 47EAD830ADF3CCE22244EE89 /* Pods-NeuroID.staging.xcconfig */;
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_IDENTITY = "";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Automatic;
```
Comment (kevinsites) [inline NeuroID.xcodeproj/project.pbxproj:834]: I thought we were going away from automatic?

**Review by kishisaka-nid — DISMISSED** [review]
LGTM 🚢

**Review by kevinsites — DISMISSED** [review]

**Review by priya-xavier — APPROVED** [review]

**Pull request merged**